Adyen/Model/Payments/ThreeDSecureData.cs

```diff
@@ -64,11 +66,12 @@ public enum AuthenticationResponseEnum
 
         }
 
+
```

Comment:
can we get rid of this additional blank line?

Reply:
and fixed